src/hooks/useConstitutionLinter.ts

```
@@ -0,0 +1,116 @@
+            if (insertError || !docRecord) throw insertError;
+
+            // 3. Trigger Edge Function for AI Linting
+            setIsUploading(false);
+            setIsAnalyzing(true);
+
+            const { data: fnData, error: fnError } = await supabase.functions.invoke('lint-constitution', {
+                body: { document_id: docRecord.id, file_url: publicUrl }
+            });
+
+            if (fnError) throw fnError;
+            if (fnData.error) throw new Error(fnData.error);
```

ESLint

[error] Replace `············` with `······`

(prettier/prettier)